### DISCUSSION TITLE
Un-Discoverable Devices Set to Online

### DISCUSSION THREAD

**Discussion by anjota80** [netalertx/NetAlertX#615] [Ideas]
I have several unmanaged switches (non-IP) added to my network. However, they show as offline in My Devices. I think it would be best to show Un-Discoverable devices as online rather than offline. Perhaps a future enhancement to the build?

![2024-04-02_22-42-12](https://github.com/jokob-sk/Pi.Alert/assets/53611664/8c066521-b693-4120-ae01-52ca5f1a0f5e)

![2024-04-02_22-40-52](https://github.com/jokob-sk/Pi.Alert/assets/53611664/97ec9086-c477-4c8a-b9e1-1b763c788649)

**Comment by jokob-sk** [comment]
Covered in #617
